node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -206,7 +198,7 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
       // Get all of our cached headers in case of a reorg
       val cachedHeaders =
         blockchains.flatMap(_.headers).map(_.hashBE.flip)
-      peerMsgSenderF.map(_.sendGetHeadersMessage(cachedHeaders))
+      peerMsgSender.sendGetHeadersMessage(cachedHeaders)
```

Comment:
This returns a `Future`, can we move this up into the for comprehension? I.e. 

```scala
// Get all of our cached headers in case of a reorg
cachedHeaders = blockchains.flatMap(_.headers).map(_.hashBE.flip)
_ <- peerMsgSender.sendGetHeadersMessage(cachedHeaders)
 ```